addon/bin/sign

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+const version = require('../package.json').version;
+const resolve = require('path').resolve;
+const xpiPath = resolve(`dist/testpilot-addon-${version}.xpi`);
```

Comment:
Ahh I didn't realize we were on that old of a node version. I'll pull the template strings out.

> Do we also need to bump the version every time we want to sign it?
yes. I tried to get a `--silent` option upstream but it doesn't look like that will happen.
I wrote a wrapper `bin/sign` in order to catch those errors so we could still build, but I'm thinking it may be better to just check the version instead of rebuilding.

I'll bump jpm version in the package.json as well.